langchain-core/src/runnables/passthrough.ts

```diff
@@ -94,12 +94,12 @@ export class RunnablePassthrough<RunInput> extends Runnable<
    * });
    * ```
    */
-  static assign(
-    // eslint-disable-next-line @typescript-eslint/no-explicit-any
-    mapping: RunnableMapLike<Record<string, unknown>, Record<string, unknown>>
-  ): RunnableAssign<Record<string, unknown>, Record<string, unknown>> {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  static assign<RunInput extends Record<string, any>, RunOutput extends Record<string, any>>(
```

Comment:
Any reason you switched `Record<string, unknown>` to `Record<string, any>`? If possible I'd prefer to keep `unknown` over `any`, but if that was giving issues it's not too big a deal.

Reply:
No particular reason, I just wanted to align it with the other runnables that use `Record<string, any>`. I switched it back to `Record<string, unknown>`, it shouldn't cause any issues.